calitp/auth.py

```diff
@@ -0,0 +1,35 @@
+import os
+from typing import Sequence
+
+import google_crc32c
+from google.cloud import secretmanager
+
+AUTH_KEYS_ENV_VAR = "CALITP_AUTH_KEYS"
+if AUTH_KEYS_ENV_VAR in os.environ:
+    DEFAULT_AUTH_KEYS = tuple(os.environ[AUTH_KEYS_ENV_VAR].split(","))
+else:
+    DEFAULT_AUTH_KEYS = (
+        "AC_TRANSIT_API_KEY",
+        "AMTRAK_GTFS_URL",
+        "CULVER_CITY_API_KEY",
+        "MTC_511_API_KEY",
+        "SD_MTS_SA_API_KEY",
+        "SD_MTS_VP_TU_API_KEY",
+        "SWIFTLY_AUTHORIZATION_KEY_CALITP",
+        "WEHO_RT_KEY",
+    )
+
+
+def load_secrets(keys: Sequence[str] = DEFAULT_AUTH_KEYS, secret_client=secretmanager.SecretManagerServiceClient()):
+    for key in keys:
+        if key not in os.environ:
```

Comment:
confused by this.... if key is locally available, what happens?

Reply:
I didn't want to override anything already in the environ. This should make it easier to migrate keys by just changing the Secret and then removing the env var in Composer.